### PR TITLE
sys-block/arcconf: Bump to version 2.06

### PR DESCRIPTION
**Pull request by ConiKost**
sys-block/arcconf: Bump to version 2.06

Also added 2.01, as it's the latest release, to support x86.
Added Microsemi license, as Adaptec-EULA license is deprecated.
Turned fetch restriction on, I don't think, we should download automatically..
Dropped old eBuilds

### CODE REVIEW

File: sys-block/arcconf/arcconf-2.01.22270.ebuild [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2018 Gentoo Foundation
```
Comment (mgorny) [inline sys-block/arcconf/arcconf-2.01.22270.ebuild]: Could you please split those two bumps into separate commits, please?

File: sys-block/arcconf/arcconf-2.01.22270.ebuild [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+pkg_setup() {
+	# CONFIG_HARDENED_USERCOPY_PAGESPAN makes ARCCONF segault
+	if linux-info_get_any_version && linux_config_src_exists ; then
+		CONFIG_CHECK="!HARDENED_USERCOPY_PAGESPAN"
+		check_extra_config
+	fi
+}
+
+src_install() {
+	exeinto /opt/arcconf
```
Comment (mgorny) [inline sys-block/arcconf/arcconf-2.01.22270.ebuild]: Do you really need to create the whole directory for one file? Also if it's fully standalone, it's fine to install it to `/usr`.

Comment (ConiKost) [reply sys-block/arcconf/arcconf-2.01.22270.ebuild]: Okay, I've changed it to ```/usr/bin```. AFAIR, someone told me not to install binary only packages in ```/usr/bin```.

File: sys-block/arcconf/arcconf-2.01.22270.ebuild (hunk 1 above)
Comment (ConiKost) [reply sys-block/arcconf/arcconf-2.01.22270.ebuild]: Done.

File: sys-block/arcconf/arcconf-2.01.22270.ebuild (hunk 2 above)
Comment (mgorny) [reply sys-block/arcconf/arcconf-2.01.22270.ebuild]: That's a common misconception. `/opt` is meant for packages which don't follow FHS, and e.g. use Windows-style install layout (data located in special directories relative to executable. However, even some of those packages end up in `/usr/lib*/foo` (web browsers being the most common example).

**Comment by ConiKost** [comment]
x( Thanks, fixed.

File: sys-block/arcconf/arcconf-2.01.22270.ebuild (hunk 2 above)
Comment (ConiKost) [reply sys-block/arcconf/arcconf-2.01.22270.ebuild]: Thank you very much!
